lib/auth/tls_test.go

```diff
@@ -1184,20 +1186,6 @@ func TestPasswordCRUD(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestTokens(t *testing.T) {
```

Comment:
Happy to keep this around until GenerateToken is removed.

Reply:
It's a little hard to keep this test around since I've stripped the client method out. I feel that this is already covered by `TestTokensCRUD` - if you don't agree, I'm happy to make this test directly invoke the gRPC handler perhaps...

Reply:
Sounds good to me, just double-checking.